src/main/drivers/pwm_output.h

```diff
@@ -18,10 +18,12 @@
 #pragma once
 
 typedef enum {
+#ifndef BRUSHED_MOTORS
```

Comment:
This will break CLI and Configurator compatibility. It's better to keep the full enum but make sure we can't select anything else than brushed (in `validateAndFixConfig()`)